var/spack/repos/builtin/packages/py-morph-tool/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMorphTool(PythonPackage):
+    """Python morphology manipulation toolkit"""
+
+    homepage = "https://github.com/BlueBrain/morph-tool"
+    git = "https://github.com/BlueBrain/morph-tool.git"
+    pypi = "morph-tool/morph-tool-2.9.1.tar.gz"
+
+    version("develop", branch="master")
```

Comment:
Is there a reason the branch name does not match the version? 

(See docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#version-comparison).

Reply:
Probably just carelessness in the days when these recipes were written - updating the name should cause no breakage for us.